front/src/BottomBar/index.js

```diff
@@ -44,7 +44,7 @@ const getConnectionStatusBackground = type => {
  */
 
 const BottomBar = props => {
-  const { changeActivePage, activePage, connectionStatus } = props;
+  const { changeActivePage, activePage, connectionStatus, token } = props;
```

Comment:
Please add "token" to the props in the documentation

Reply:
done